lib/ione/rpc/server.rb

```diff
@@ -52,7 +52,9 @@ def handle_connection(connection)
 
       # Override this method to handle errors raised or returned by
       # {#handle_request} or any other part of the request handling (for example
-      # the response encoding).
+      # the response encoding). In the event that {#handle_request} completed
+      # successfully, # the previous response will contain that result, otherwise
+      # it will be nil.
```

Comment:
I think "previous response" is a confusing name, I think it's better to just call it "response".

There's also a stray # in the middle of the second line.